horace_core/sqw/PixelData/@MultipixBase/MultipixBase.m

```diff
@@ -0,0 +1,289 @@
+classdef MultipixBase < serializable
+    % The class-base for helper classes, used to keep information about multiple
+    % pixels datasets before they are combined together.
+    %
+    properties(Dependent)
+        nfiles;       % number of files or objects contributing into final
+        %               result
+        infiles;      % cellarray of filenames or objects to combine.
+        %
+        num_pixels;   % total number of pixels to combine in all
+        %                contributing pixels datasets or files
+        npix_each_file; % array defining numbers of pixels stored in each
+        %                contributing file or datasets
+        %
+        % number of bins (number of elements in npix array) in the
+        % contributing sqw(tmp) files. Should be the same for all
+        % files to be able to combine them together
+        nbins;
+
+        pix_range    % Global range of all pixels, intended for combining
+        data_range   % Global range of all pixel data, i.e. coordinates, signal error and other pixel parameters
+
+        %   run_label   Indicates how to re-label the run index (pix(5,...)
+        %          'fileno'      relabel run index as the index of the file
+        %                        in the list infiles
+        %          'nochange'    use the run index as in the input file
+        %                        numeric array  offset run numbers for ith
+        %                        file by ith element of the array
+        %          This option exists to deal with three limiting cases:
+        %      (1) The run index is already written to the files correctly indexed into the header
+        %          e.g. as when temporary files have been written during cut_sqw
+        %      (2) There is one file per run, and the run index in the header block is the file
+        %          index e.g. as in the creating of the master sqw file
+        %      (3) The files correspond to several runs in general, which need to
+        %          be offset to give the run indices into the collective list of run parameters
+        run_label;
+
+        % numbers of files used as run_label for pixels if relabel_with_fnum
+        % and change_fileno are set to true
+        filenum
+        %
+        % true if pixel id from each contributing file should be replaced by contributing file number
+        relabel_with_fnum;
+        % true if pixel id for each pixel from contributing files should be changed.
+        change_fileno
+    end
+    properties(Dependent,Hidden)
+        % The property, which describes the pixel data layout on disk or in
+        % memory and all additional properties describing pix array
+        metadata;
+        data_wrap;
+        % PixelDataBase interface
+        full_filename
+        is_filebacked
+        % the property here to support PixelData interface. Never false, as
+        % this kind of data should be never (knowingly) misaligned
```

Comment:
And if it is `true`? What happens? Is it ignored? Does it error?

Reply:
Noting below that this is forcibly `false`. Should probably emphasise this here if that's the case. Also, the comment says "never false" where it should say "never true" or "always false"